psalm.xml.dist

```diff
@@ -10,6 +10,7 @@
         <ignoreFiles>
             <directory name="var"/>
             <directory name="vendor"/>
+            <directory name="tests/TraceContext/W3CTestService"/>
```

Comment:
Excluding the test service directory here will avoid some unwanted psalm errors. Some files used by the test service are dynamically created so a psalm test run will show errors for these files that don't yet exist.